install_config/install/advanced_install.adoc

```diff
@@ -959,6 +959,9 @@ openshift_master_cluster_method=native
 openshift_master_cluster_hostname=openshift-cluster.example.com
 openshift_master_cluster_public_hostname=openshift-cluster.example.com
 
+# apply updated node defaults
+openshift_node_kubelet_args={'pods-per-core': ['10'], 'max-pods': ['250'], 'image-gc-high-threshold': ['90'], 'image-gc-low-threshold': ['80']}
```

Comment:
That's the purpose of this:  https://bugzilla.redhat.com/show_bug.cgi?id=1371309
But we're sort of all over the place as to where we file stuff because the rules are unclear.